tests/std/tests/P1208R6_source_location/test.cpp

```diff
@@ -184,12 +201,21 @@ constexpr void function_template_test() {
 constexpr bool test() {
     copy_test();
     local_test();
+#ifndef __clang__ // TRANSITION, LLVM-56379
     argument_test(__LINE__, 5);
+#endif // __clang__
+#ifdef __clang__
+    const auto loc = source_location::current();
+    argument_test(__LINE__ - 1, 22, loc);
+#else // ^^^ defined(__clang__) / !defined(__clang__)

```

Comment:
No change requested: Missing `vvv` arrows. This isn't worth resetting testing, I'll clean it up later.